config/my-e-car.json

```diff
@@ -11,283 +11,155 @@
-                    },
-                    {
-                        "interval": 60,
-                        "rate": 0.95,
-                        "start": 1440
-                    }
-                ],
-                "per_km_pricing": [
-                    {
-                        "start": 0,
-                        "end": 100,
-                        "interval": 1,
-                        "rate": 0.26
-                    },
-                    {
-                        "start": 100,
-                        "interval": 1,
-                        "rate": 0.22
-                    }
-                ],
-                "plan_id": "micro",
+                "name": "A-e Renault ZOE bis 140km - Nacht pro Stunde",
+                "per_min_pricing": [ { "start": 0, "end": 420, "interval": 60, "rate": 0 } ],
+                "per_km_pricing": [ { "start": 0, "end": 100, "interval": 1, "rate": 0.26 }, { "start": 101, "interval": 1,"rate": 0.22} ],
+                "plan_id": "micro_hour_nighttime",
                 "price": 0,
                 "url": "https://www.my-e-car.de/wp-content/uploads/sites/3/2023/06/myD-PK-Preise-Tarife-Zusatzkosten.pdf"
             },
             {
                 "currency": "EUR",
-                "description": "Klassik-Tarif: Am ersten Tag 2,00€/h, am 2. Tag 1,20€/h. Nachtpreis zwischen 0h und 7h: 0€. 0,26€/km bis 100km, danach 0,22€/km. Zuzüglich Grundgebühr. Alle Angaben ohne Gewähr.",
+                "description": "Klassik-Tarif. Für jede Buchung setzen sich die Nutzungskosten zusammen aus Zeittarif + Kilometertarif.",
                 "is_taxable": false,
-                "name": "B 5-Sitzer-Kleinwagen (KLASSIK/AKTIV)",
-                "per_min_pricing": [
-                    {
-                        "start": 0,
-                        "end": 1020,
-                        "interval": 60,
-                        "rate": 2.0
-                    },
-                    {
-                        "start": 1020,
-                        "end": 1440,
-                        "interval": 60,
-                        "rate": 0
-                    },
-                    {
-                        "interval": 60,
-                        "rate": 1.20,
-                        "start": 1440
-                    }
-                ],
-                "per_km_pricing": [
-                    {
-                        "start": 0,
-                        "end": 100,
-                        "interval": 1,
-                        "rate": 0.26
-                    },
-                    {
-                        "start": 100,
-                        "interval": 1,
-                        "rate": 0.22
-                    }
-                ],
-                "plan_id": "mini",
+                "name": "A-e Renault ZOE bis 140km - Tag pro Stunde",
+                "per_min_pricing": [ { "start": 0, "end": 1020, "interval": 60, "rate": 1.50 }, { "start": 1021, "interval": 60, "rate": 0.95 } ],
```

Comment:
Can you revert to the old formatting/indentation style? This way, your commits only contain actual changes, which makes them more understandable later on.

(Maybe there's a good reason that I'm not aware of to change the formatting here. Ignore my comment in this case.)

Reply:
I changed the formatting of stadtmobil_* and my-e-car because otherwise the pricing descriptions become very lengthy.